packages/kbn-securitysolution-exception-list-components/src/exception_items/index.tsx

```diff
@@ -93,6 +95,7 @@ const ExceptionItemsComponent: FC<ExceptionItemsProps> = ({
         buttonText={emptyViewerButtonText}
         body={emptyViewerBody}
         onCreateExceptionListItem={onCreateExceptionListItem}
+        listType={listType === ExceptionListTypeEnum.ENDPOINT ? ListTypeText.ENDPOINT : undefined}
```

Comment:
I believe we don't need this change in the `kbn-package` as we can have the `emptyViewerButtonText` and the `emptyViewerBody` props so we can from the parent component pass the text based on this condition
 `listType === ExceptionListTypeEnum.ENDPOINT ? ListTypeText.ENDPOINT : undefined`

So if we change `x-pack/plugins/security_solution/public/exceptions/components/exceptions_list_card/index.tsx` 
```
 <ListExceptionItems
                  emptyViewerBody={newBody} <== here
                  emptyViewerTitle={newTitle} <== here
                  isReadOnly={readOnly}
                  exceptions={exceptions}
                  listType={exceptionsList.type as ExceptionListTypeEnum}
                  pagination={pagination}
                  hideUtility
                  viewerStatus={exceptionViewerStatus}
                  ruleReferences={ruleReferences}
                  onDeleteException={onDeleteException}
                  onEditExceptionItem={onEditExceptionItem}
                  onPaginationChange={onPaginationChange}
                  onCreateExceptionListItem={onAddExceptionClick}
                  lastUpdated={null}
                />

```